### ISSUE TITLE
pkgdown changed build_favicon() to build_favicons()

### ISSUE THREAD

**Issue by jennybc** [r-lib/usethis#844]
What are the implications for `usethis::use_pkgdown()` and `use_pkgdown_travis()`?

**Comment by andrie** [comment]
With both `pkgdown` and `usethis` installed from current CRAN versions, I get this error message:

```
> use_pkgdown_travis()
✔ Setting active project to 'C:/Users/apdev/Documents/github/R-packages/experimental/gcalendr/gcalendr'
`build_favicon()` is deprecated as of pkgdown 1.4.0. Please use `build_favicons()` instead.
-- Building favicons -----------------------------------------------------------
Error in build_favicons(pkg, overwrite) : 
  argument "overwrite" is missing, with no default
```

My `session_info()`:

```- Session info --------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.6.1 (2019-07-05)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United Kingdom.1252 
 ctype    English_United Kingdom.1252 
 tz       Europe/London               
 date     2019-09-06                  

- Packages ------------------------------------------------------------------------------------------
 package     * version    date       lib source                          
 assertthat    0.2.1      2019-03-21 [1] CRAN (R 3.6.0)                  
 backports     1.1.4      2019-04-10 [1] CRAN (R 3.6.0)                  
 cli           1.1.0      2019-03-19 [1] CRAN (R 3.6.0)                  
 clisymbols    1.2.0      2017-05-21 [1] CRAN (R 3.5.1)                  
 crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.1)                  
 curl          4.0        2019-07-22 [1] CRAN (R 3.6.1)                  
 desc        * 1.2.0      2018-05-01 [1] CRAN (R 3.5.1)                  
 digest        0.6.20     2019-07-04 [1] CRAN (R 3.6.1)                  
 dplyr         0.8.3      2019-07-04 [1] CRAN (R 3.6.1)                  
 evaluate      0.14       2019-05-28 [1] CRAN (R 3.6.0)                  
 fs            1.3.1      2019-05-06 [1] CRAN (R 3.6.0)                  
 gargle        0.3.1      2019-07-26 [1] CRAN (R 3.6.1)                  
 gcalendr      0.1.2.9000 2019-06-11 [1] local                           
 glue          1.3.1      2019-03-12 [1] CRAN (R 3.6.0)                  
 here          0.1        2017-05-28 [1] CRAN (R 3.4.1)                  
 htmltools     0.3.6      2017-04-28 [1] CRAN (R 3.5.1)                  
 httr          1.4.1      2019-08-05 [1] CRAN (R 3.6.1)                  
 knitr         1.24       2019-08-08 [1] CRAN (R 3.6.1)                  
 lubridate     1.7.4      2018-04-11 [1] CRAN (R 3.5.1)                  
 magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.1)                  
 MASS          7.3-51.4   2019-03-31 [2] CRAN (R 3.6.1)                  
 memoise       1.1.0      2017-04-21 [1] CRAN (R 3.5.1)                  
 packrat       0.5.0-21   2019-09-02 [1] Github (rstudio/packrat@29a4fdc)
 pillar        1.4.2      2019-06-29 [1] CRAN (R 3.6.0)                  
 pkgconfig     2.0.2      2018-08-16 [1] CRAN (R 3.5.1)                  
 pkgdown       1.4.0      2019-09-03 [1] CRAN (R 3.6.1)                  
 purrr         0.3.2      2019-03-15 [1] CRAN (R 3.6.0)                  
 R6            2.4.0      2019-02-14 [1] CRAN (R 3.6.0)                  
 Rcpp          1.0.2      2019-07-25 [1] CRAN (R 3.6.1)                  
 rematch2      2.1.0      2019-07-11 [1] CRAN (R 3.6.1)                  
 reprex      * 0.3.0      2019-05-16 [1] CRAN (R 3.6.0)                  
 rlang         0.4.0      2019-06-25 [1] CRAN (R 3.6.1)                  
 rmarkdown     1.15       2019-08-21 [1] CRAN (R 3.6.1)                  
 rprojroot     1.3-2      2018-01-03 [1] CRAN (R 3.5.1)                  
 rstudioapi    0.10       2019-03-19 [1] CRAN (R 3.6.0)                  
 sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.5.1)                  
 stringi       1.4.3      2019-03-12 [1] CRAN (R 3.6.0)                  
 stringr       1.4.0      2019-02-10 [1] CRAN (R 3.6.0)                  
 tibble        2.1.3      2019-06-06 [1] CRAN (R 3.6.0)                  
 tidyselect    0.2.5      2018-10-11 [1] CRAN (R 3.5.1)                  
 usethis     * 1.5.1      2019-07-04 [1] CRAN (R 3.6.1)                  
 withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.1)                  
 xaringan      0.11       2019-07-12 [1] CRAN (R 3.6.1)                  
 xfun          0.9        2019-08-21 [1] CRAN (R 3.6.1)                  
 yaml          2.2.0      2018-07-25 [1] CRAN (R 3.5.1)                  
```

**Comment by hadley** [comment]
Fixed in #882
